internal/bminventory/inventory.go

```diff
@@ -3543,7 +3543,7 @@ func (b *bareMetalInventory) DownloadClusterFiles(ctx context.Context, params in
 		return filemiddleware.NewResponder(installer.NewDownloadClusterFilesOK().WithPayload(ioutil.NopCloser(strings.NewReader(cfg))), params.FileName, int64(len(cfg)))
 	}
 
-	if funk.Contains(clusterPkg.ClusterOwnerFileNames, params.FileName) {
+	if funk.Contains(clusterPkg.ClusterOwnerFileNames, params.FileName) && b.authHandler.AuthType() == auth.TypeRHSSO {
```

Comment:
Could you short docstring explaining why we check that `AuthType() == rhsso`?  I think this check is important (and weird) enough to warrant a short comment :)